htdocs/fourn/class/fournisseur.commande.class.php

```diff
@@ -2104,7 +2104,10 @@ function setStatus($user,$status)
         $resql = $this->db->query($sql);
         if ($resql)
         {
-            //TODO: Add trigger for status modification
+            // Call trigger
+            $result=$this->call_trigger('ORDER_SUPPLIER_SET_STATUS',$user);
```

Comment:
To be consistent with rest of code, triggers name must depends on status and use already existing value.
For exemple:
ORDER_SUPPLIER_CANCEL
ORDER_SUPPLIER_REFUSE
ORDER_SUPPLIER_APPROVE
ORDER_SUPPLIER_CLASSIFY_BILLED